tests/series/test_utf8_ops.py

```diff
@@ -294,3 +294,19 @@ def test_series_utf8_rstrip(data, expected) -> None:
     s = Series.from_arrow(pa.array(data))
     result = s.str.rstrip()
     assert result.to_pylist() == expected
+
+
+@pytest.mark.parametrize(
+    ["data", "expected"],
+    [
+        (["abc", "def", "ghi"], ["cba", "fed", "ihg"]),
+        # With at least one null
+        (["abc", None, "def", "ghi"], ["cba", None, "fed", "ihg"]),
+        # With all nulls
+        ([None] * 4, [None] * 4),
```

Comment:
It would be good to add a case of emojis and foreign characters

Reply:
Added tests for both